_queue/template.robot

```diff
@@ -0,0 +1,48 @@
+*** Settings ***
```

Comment:
Ok, makes sense.  I think queue is a bit confusing, what about calling it:

- `templates`
- `unimplemented`
- `in_progress`
- `disabled`